### PR TITLE
Add images to UG (View, Close, Help)

### PR DESCRIPTION
**Pull request by itsme-zeix**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -765,6 +765,17 @@ view <INDEX>
 - **On Success:**
     - A split view opens showing the selected client's complete information
     - The main list remains visible and functional on the left while detailed information appears on the right
+    - The image below the GUI before and after using `view 1`:
```
Comment (ZShunRen) [inline docs/UserGuide.md:768]:
```suggestion
    - The images below show the GUI before and after using `view 1`:
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -794,6 +805,17 @@ close
 - **On Success:**
     - The split view closes.
     - The main list view returns to full width.
+    - The image below the GUI before and after using `close`:
```
Comment (ZShunRen) [inline docs/UserGuide.md:808]:
```suggestion
    - The images below show the GUI before and after using `close`:
```

**Review by ZShunRen — APPROVED** [review]
LGTM, just mind ur grammar mate

File: docs/UserGuide.md (hunk 2 above)
Comment (itsme-zeix) [reply docs/UserGuide.md:808]: Nice catch!

**Review by colinhia — APPROVED** [review]
LGTM!

**Pull request merged**